Leanplum-SDK/Classes/LPRequestStorage.m

```diff
@@ -168,21 +200,9 @@ - (NSArray *)popAllRequests
                 LPLog(LPDebug, @"Deleted stored requests.");
             }
         }
-
-        // For compatibility with older SDKs.
-        // TODO: Remove in April 2015.
-        NSInteger count = [_defaults integerForKey:LEANPLUM_DEFAULTS_COUNT_KEY];
-        if (count > 0) {
-            [_defaults removeObjectForKey:LEANPLUM_DEFAULTS_COUNT_KEY];
-            for (NSInteger i = 0; i < count; i++) {
-                NSString* itemKey = [LPRequestStorage itemKeyForIndex:i];
-                [_defaults removeObjectForKey:itemKey];
-            }
-            [_defaults synchronize];
-        }
-
     }
-    return result;
+    NSLog(@"pop: %@", requests);
```

Comment:
Intentional NSLog?
I'd say we should at least add `Leanplum - ` prefix, or use LPLOG